lib/tabulation/src/Data/Tabulation.hs

```diff
@@ -18,4 +19,8 @@ class HasFields a where
   traverseWithField t r = tabulateFieldsA (\f -> t f (indexField r f))
   indexField :: a -> Field a x -> x
   indexField a f = a ^. fieldLens f
-  {-# MINIMAL fieldLens, tabulateFieldsA #-}
\ No newline at end of file
+  {-# MINIMAL fieldLens, tabulateFieldsA #-}
+
+-- | A wrapper to allow mixing/reusing a non-HDK associated GADT with the higher kinded data record
```

Comment:
```suggestion
-- | A wrapper to allow mixing/reusing a non-HKD associated GADT with the higher kinded data record
```
Although seeing HDK in two places makes me think I might be wrong and this is actually a term I'm missing.

Reply:
Oops, typo and copy-pasta